io/net/sriov_device_test.py.data/README.txt

```diff
@@ -10,6 +11,32 @@ ipaddr = ip address for ip configuration in sriov logical interface.
 netmask = specify the netmask for ip configuration.
 peer_ip = specify peer ip address.
 mac_id = mac address for sriov logical device.
+max_sriov_ports = Specify the max number of logical SRIOV ports to be created.
+
+Note1:
+-----
+when YAML parameters given with space seperated, the script creates "N" number of 
+logical ports with Network settings assocaited with it.
+
+Ex: To create 2 SRIOV logical ports coming from each port when card having 2 physical port.
+
+sriov_adapter: "U78CD.001.FZHAE88-P2-C6 U78CD.001.FZHAE88-P2-C6"
+sriov_port: "0 1"
```

Comment:
@FarooqAbdulla02 why to give "0 1".
pass only one number like if you want 5 logical port then just give 5 in yaml should be fine.
handle the ID like 0 1 2 3 4 5 in script wit for loop

Reply:
@Naresh-ibm yes, you can specify single port like "0", and logical ports are only created for that port.
here are the sample logs


[root@ltcden4-lp2-test net]# cat sriov_device_test.py.data/sriov_device_test.yaml
hmc_username: "xxxx"
hmc_pwd: "xxxx"
sriov_adapter: "U78CD.001.FZHAE88-P2-C6"
sriov_port: "0"
ipaddr: "192.168.180.1 "
netmasks: "255.255.255.0 "
peer_ip: "192.168.180.2 "
mac_id: "16:16:3e:77:93:c9 "
max_sriov_ports: 4 



[root@ltcden4-lp2-test net]# avocado run sriov_device_test.py:NetworkSriovDevice.test_add_max_logical_devices -m sriov_device_test.py.data/sriov_device_test.yaml
JOB ID     : 6043ce755a6f95fbc0161add4098ed36c9ab2250
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2023-08-21T05.46-6043ce7/job.log
 (1/1) sriov_device_test.py:NetworkSriovDevice.test_add_max_logical_devices;run-7ef9: STARTED
 (1/1) sriov_device_test.py:NetworkSriovDevice.test_add_max_logical_devices;run-7ef9: PASS (53.44 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2023-08-21T05.46-6043ce7/results.html

Note : All the above 4 logical SRIOV ports are created only with port 0 here

[root@ltcden4-lp2-test net]# avocado run sriov_device_test.py:NetworkSriovDevice.test_remove_max_logical_devices -m sriov_device_test.py.data/sriov_device_test.yaml
JOB ID     : 035a6dd1e735d990565e590d6d44724654ab6358
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2023-08-21T05.49-035a6dd/job.log
 (1/1) sriov_device_test.py:NetworkSriovDevice.test_remove_max_logical_devices;run-7ef9: STARTED
 (1/1) sriov_device_test.py:NetworkSriovDevice.test_remove_max_logical_devices;run-7ef9: PASS (68.72 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2023-08-21T05.49-035a6dd/results.html
JOB TIME   : 83.50 s

Reply:
then please update same README then

Reply:
@Naresh-ibm in NIC /virtual, ports are refered from 0 for port1 and 1 for port2 always.. this has been followed always